### PR TITLE
33 add actions costs

### LINKED ISSUE #33: Add actions costs
UI update

### PR DESCRIPTION
**Pull request by LavergneC**

### CODE REVIEW

File: Classic_RPG/src/game_management/Interfaces/GUI.java [hunk 1]
```diff
@@ -290,38 +276,118 @@ public void actionPerformed(ActionEvent arg0) {
...
+		costs_panel_rest.add(lblNewLabel_5);
+
+		JLabel lblNewLabel_1_2 = new JLabel("New label");
+		costs_panel_rest.add(lblNewLabel_1_2);
+
+		JLabel lblNewLabel_2_2 = new JLabel("New label");
+		costs_panel_rest.add(lblNewLabel_2_2);
+
+		JLabel lblNewLabel_3_2 = new JLabel("New label");
+		costs_panel_rest.add(lblNewLabel_3_2);
```
Comment (nicolasGibaud7) [inline Classic_RPG/src/game_management/Interfaces/GUI.java]: Think to delete them

File: Classic_RPG/src/game_management/Interfaces/GUI.java [hunk 2]
```diff
@@ -290,38 +276,118 @@ public void actionPerformed(ActionEvent arg0) {
...
+		costs_panel_def.add(lblNewLabel_4);
+
+		JLabel lblNewLabel_1_1 = new JLabel("New label");
+		costs_panel_def.add(lblNewLabel_1_1);
+
+		JLabel lblNewLabel_2_1 = new JLabel("New label");
+		costs_panel_def.add(lblNewLabel_2_1);
+
+		JLabel lblNewLabel_3_1 = new JLabel("New label");
+		costs_panel_def.add(lblNewLabel_3_1);
```
Comment (nicolasGibaud7) [inline Classic_RPG/src/game_management/Interfaces/GUI.java]: Think to delete

**Review by nicolasGibaud7 — CHANGES_REQUESTED** [review]
just useless labels to delete before merge

File: Classic_RPG/src/game_management/Interfaces/GUI.java (hunk 2 above)
Comment (LavergneC) [reply Classic_RPG/src/game_management/Interfaces/GUI.java]: done

**Pull request merged**